### ISSUE TITLE
Support Multi-Line COMPOPTS files

### ISSUE THREAD

**Issue by buddha314** [chapel-lang/chapel#7500]
As applications get more complicated, so do the compiler commands.  Forcing all compiler commands on to one line makes them illegible (and I'm the NLP guy, mind you...)

COMPOPTS during `start_test` should support multi-line commands as in

```
-M../src \\
-L/usr/local/Cellar/openblas/0.2.20/lib -lblas
```

That is, assuming `start_test` will be supporting end-user testing requirements, as discussed in #7495

**Comment by bradcray** [comment]
I think this is a valid feature request whether or not start_test is intended for end-users.  If we do it for COMPOPTS/.compopts files, we should also do it for EXECOPTS/.execopts.

**Comment by ben-albrecht** [comment]
Just to be clear, you can do this today with `<your favorite interpreter>` following @ronawho's previous [example](https://github.com/chapel-lang/chapel/issues/7495#issuecomment-332655348), e.g.

```python
#!/usr/bin/env python

# COMPOPTS file

from __future__ import print_function

print('-M../src '
      '-L/usr/local/Cellar/openblas/0.2.20/lib -lblas')
```

**Comment by bradcray** [comment]
Still seems like it'd be a nice convenience function for the text-only compopts/execopts cases for those who don't get excited about writing scripts.

**Comment by buddha314** [comment]
Just to be clear, my favorite interpreter is C3P0

**Comment by ben-albrecht** [comment]
> Still seems like it'd be a nice convenience function for the text-only compopts/execopts cases for those who don't get excited about writing scripts.

I agree - just wanted to make it clear for anyone else who stumbles upon this issue looking for this functionality.

**Comment by buddha314** [comment]
> Still seems like it'd be a nice convenience function for the text-only compopts/execopts cases for those who don't get excited about writing scripts.

And I agree except that I come from Hollywood where everyone is writing a script.  Got six lines of python from the guy who sold me a taco off the truck this morning..
